javaCore/src/main/java/com/hamdan/forzenbook/java/core/utils/OnDialogSubmission.java

```diff
@@ -0,0 +1,5 @@
+package com.hamdan.forzenbook.java.core.utils;
+
+public interface OnDialogSubmission {
```

Comment:
Instead of a bunch of one off interfaces it's pretty common to see `Runnable` used

Reply:
ok runnable is pretty nice